apps/chrome-devtools/src/extension/background.ts

```diff
@@ -18,6 +32,7 @@ const WHITELISTED_HOSTS_KEY: typeof WhitelistedHostsKey = 'WHITELISTED_HOSTS';
  * @param message
  */
 const isOtterDebugMessage = (message: any): message is OtterMessage => {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access -- message is of type `any`
   return message?.type === postMessageType;
```

Comment:
wouldn't this be cleaner ?
```suggestion
  return (message as OtterMessage)?.type === postMessageType;
```